src/main/java/uk/gov/ons/fwmt/job_service/service/impl/CSVParsingServiceImpl.java

```diff
@@ -130,47 +135,59 @@ public LocalDate convertToLFSDate(String stage) throws FWMTCommonException {
       final FieldPeriodDto fieldPeriod = existsByFieldperiod.get();
```

Comment:
convertToGFFDate and convertToLFSDate have duplicate code, merge to one call.

Reply:
These have been merged